src/instruments/src/EFB/Dashboard/Widgets/WeatherWidget.tsx

```diff
@@ -115,100 +118,135 @@ const WeatherWidget = (props: WeatherWidgetProps) => {
+                                </button>
                             </div>
                         </div>
+                        {showMetar === 'DISABLED'
+                            ? (
+                                <>
+                                    <div className="grid grid-cols-2 h-40">
+                                        <div className="justify-left text-center text-lg">
+                                            <div className="flex justify-center">
+                                                <IconGauge className="mb-2" size={35} stroke={1.5} strokeLinejoin="miter" />
+                                            </div>
+                                            {metar.raw_text ? (
+                                                <>
+                                                    {metar.barometer.mb.toFixed(0)}
+                                                    {' '}
+                                                    mb
+                                                </>
+                                            ) : (
+                                                <>
+                                                    N/A
+                                                </>
+                                            )}
+                                        </div>
+                                        <div className="justify-left text-center text-lg">
+                                            <div className="flex justify-center">
+                                                <IconWind className="mb-2" size={35} stroke={1.5} strokeLinejoin="miter" />
+                                            </div>
+                                            {metar.raw_text
+                                                ? (
+                                                    <>
+                                                        {metar.wind.degrees.toFixed(0)}
+                                                        {' '}
+                                                        <IconPoint
+                                                            className="inline-block -mx-1 -mt-3"
+                                                            size={20}
+                                                            stroke={2}
+                                                            strokeLinejoin="miter"
+                                                        />
+                                                        {' '}
+                                                        /
+                                                        {' '}
+                                                        {metar.wind.speed_kts.toFixed(0)}
+                                                        {' '}
+                                                        kts
+                                                    </>
+                                                ) : 'N/A'}
```

Comment:
It seems like this code segments similar to this one are recurrent throughout this file; consider extracting this into a component for conciseness

Reply:
That is basically as it was before - I just fixed a small bug by using metar.raw_text as condition to display value or N/A.
As these section are not identical but only similar it is probably best to leave them for now with you already working on v3.